utils/build.ps1

```diff
@@ -1227,8 +1252,13 @@ function Get-Dependencies {
 
     if ($EnableCaching) {
       $SCCache = Get-SCCache
-      DownloadAndVerify $SCCache.URL "$BinaryCache\sccache-$SCCacheVersion.zip" $SCCache.SHA256
-      Expand-ZipFile sccache-$SCCacheVersion.zip $BinaryCache sccache-$SCCacheVersion
+      $FileExtension = if ($SCCache.URL.EndsWith('.tar.gz')) { "tar.gz" } else { "zip" }
```

Comment:
`[System.IO.Path]::GetExtension()` is better to get the extension; that along with the inversion below should work.

```pwsh
if ($Extension -eq "zip") {
  Expand-ZipFile ...
} else {
  Expand-TapeArchive ...
}
```

Reply:
Thanks! Testing only on Windows for now because I'm not sure if this API works for URLs as well.